src/script/components/MessagesList/Message/MessageWrapper.tsx

```diff
@@ -110,6 +111,8 @@ export const MessageWrapper: React.FC<MessageParams & {hasMarker: boolean; isMes
         incomingQuote && isOutgoingQuote(incomingQuote) ? (incomingQuote as OutgoingQuote) : undefined;
 
       await messageRepository.sendTextWithLinkPreview(conversation, messageText, mentions, quote, messageId);
+    } else if (file) {
+      await messageRepository.retryUploadFile(conversation, file, firstAsset.isImage(), message.id);
```

Comment:
Can you confirm that this call will actually remove the `blob` from the database if successful?

Reply:
It doesn't, I adress it in this commit:  https://github.com/wireapp/wire-webapp/pull/14890/commits/8d84b4d2ef8dbe9c498fb0f8cb1fb5d3f4cb07af